cost-analyzer/values.yaml

```diff
@@ -409,16 +453,6 @@ prometheus:
               record: kubecost_savings_memory_allocation_bytes
               labels:
                 daemonset: "true"
-            - expr: label_replace(sum(kube_pod_status_phase{phase="Running",namespace!="kube-system"} > 0) by (pod, namespace), "pod_name", "$1", "pod", "(.+)")
```

Comment:
nice! can all recording rules go at this point, or other still being used?!

Reply:
Great question. I'm removing the ones that I know were used in Abandoned Workloads and Request Sizing. I'm not sure where the others are in use.

Reply:
I believe there's a small scale test that needs to be run to double check max() on CPU usage. I don't anticipate that that's a large query though, since we now have the 1h progressive builds :)